test/test_regenerator.rb

```diff
@@ -107,7 +107,7 @@ class TestRegenerator < JekyllUnitTest
       File.rename(@layout_path + ".tmp", @layout_path)
     end
 
-    should "handle deleted/nonexistent dependencies" do
+    should "handle deleted/non-existent dependencies" do
```

Comment:
I disagree. It's a combination of 2 words but anyway I'll revert it since I
have no interest in debating this.

On Jul 18, 2016 21:08, "Pat Hawks" <notifications@github.com> wrote:

> In test/test_regenerator.rb
> <https://github.com/jekyll/jekyll/pull/5114#discussion_r71198002>:
>
> > @@ -107,7 +107,7 @@ class TestRegenerator < JekyllUnitTest
> >        File.rename(@layout_path + ".tmp", @layout_path)
> >      end
> >
> > -    should "handle deleted/nonexistent dependencies" do
> > +    should "handle deleted/non-existent dependencies" do
>
> nonexistent <http://www.dictionary.com/browse/nonexistent> Is correct.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jekyll/jekyll/pull/5114/files/4192bf71350b77f080d9f5b66028bcb8124470e0#r71198002>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtToR7YCQUXbbP5C2k2fP1SM4p1mTks5qW8EGgaJpZM4JO6oT>
> .
>